GitUI/CommandsDialogs/BrowseDialog/FormOpenDirectory.cs

```diff
@@ -121,14 +121,13 @@ private void folderGoUpbutton_Click(object sender, EventArgs e)
 
         private void _NO_TRANSLATE_Directory_TextChanged(object sender, EventArgs e)
         {
-            try
```

Comment:
It is better to have this try-catch block in case an user has typed any path that causes an exception being thrown by DirectoryInfo. In the catch block the folderGoUpbutton should be disabled in such a case.